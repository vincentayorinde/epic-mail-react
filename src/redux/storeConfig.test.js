import store from './storeConfig';

describe('Test Store', () => {
  test('Should Test Redux Store', () => {
    expect(store).toMatchSnapshot();
  });
});
