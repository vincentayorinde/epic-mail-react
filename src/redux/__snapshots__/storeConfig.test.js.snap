// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Store Should Test Redux Store 1`] = `
Object {
  "dispatch": [Function],
  "getState": [Function],
  "replaceReducer": [Function],
  "subscribe": [Function],
  Symbol(observable): [Function],
}
`;
